### PR TITLE
Adds Google Maps

### PR DESCRIPTION
**Pull request by ariperez**
@tejans24 , my first PR in awhile.  Let the fun begin!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,36 +1,85 @@
 import React, { Component } from 'react';
-import { Grid, Navbar, Jumbotron, Button } from 'react-bootstrap';
+import { Grid, Row, Col, Navbar } from 'react-bootstrap';
```
Comment (tejans24) [inline src/App.js]: I'm not a huge fan of these grid components, but this works.

File: src/App.js [hunk 2]
```diff
@@ -1,36 +1,85 @@
...
+import { withGoogleMap, GoogleMap, Marker } from "react-google-maps";
```
Comment (tejans24) [inline src/App.js]: This looks like cool library

File: src/App.js [hunk 3]
```diff
@@ -1,36 +1,85 @@
...
 import logo from './logo.svg';
 import './App.css';
 
+
+const GettingStartedGoogleMap = withGoogleMap(props => (
+  <GoogleMap
+    ref={props.onMapLoad}
+    defaultZoom={12}
```
Comment (tejans24) [inline src/App.js]: These magic numbers need to be constants or dynamic data later on.

Comment (ariperez) [inline src/App.js]: I"m sure they will have to be dynamic based on Screen Size

File: src/App.js (hunk 1 above)
Comment (ariperez) [inline src/App.js]: What do you recommend?  Isn't it standard for all UI frameworks?

**Comment by ariperez** [comment]
Sorry, would have merged this earlier and had the next one ready, but I have to re=-enable email notifications.  Or they're going to my va? I'll figure it out and have the next one up soon.

File: src/App.js (hunk 1 above)
Comment (tejans24) [inline src/App.js]: Just css class names are the alternative.

**Comment by ariperez** [comment]
Intersting, I'll pursue this path and possibly modify if I feel the pain.

**Pull request merged**